lib/trento/clusters/commands/register_offline_cluster_host.ex

```diff
@@ -0,0 +1,19 @@
+defmodule Trento.Clusters.Commands.RegisterOfflineClusterHost do
+  @moduledoc """
+  Register a cluster node to the monitoring system.
```

Comment:
nitpick: maybe change the docstring to specify that we are talking about an offline node